app/views/users/change_email.html.erb

```diff
@@ -19,10 +19,11 @@
     <dt><%= label_tag :email_confirmation, ts("Confirm New Email") %></dt>
     <dd><%= email_field_tag :email_confirmation %></dd>
 
-    <% unless @user.recently_reset? %>
-      <dt><%= label_tag :password_check, ts("Password") %></dt>
-      <dd><%= password_field_tag :password_check%></dd>
-    <% end %>
+    <dt><%= label_tag :password_check, ts("Password") %></dt>
+    <%= password_field_tag :password_check, nil, "aria-describedby" => "password-field-description" %>

```

Comment:
The password field and the footnote still need to be wrapped in `<dd></dd>` tags to avoid invalid HTML -- sorry for being unclear!